pysat/_instrument.py

```diff
@@ -59,13 +59,10 @@ class Instrument(object):
     strict_time_flag : boolean
         If true, pysat will check data to ensure times are unique and
         monotonically increasing. (default=True)
-    multi_file_day : boolean or NoneType
+    multi_file_day : boolean
         Set to True if Instrument data files for a day are spread across
         multiple files and data for day n could be found in a file
-        with a timestamp of day n-1 or n+1.  (default=None)
-    manual_org : bool
-        if True, then pysat will look directly in pysat data directory
-        for data files and will not use default /platform/name/tag
+        with a timestamp of day n-1 or n+1.  (default=False)
     directory_format : str
```

Comment:
This and others to be updated in this pull?

Reply:
Updating with latest docstring.  Is there something else I've missed?

Reply:
I didn't see anything just now, so I think we're at least better off :P